src/v2/Components/SavedSearchAlert/DownloadAppBanner.tsx

```diff
@@ -55,7 +55,7 @@ export const DownloadAppBanner: React.FC<DownloadAppBannerProps> = ({
   return (
     <Box p={2} pb={1} backgroundColor="black5">
       <Text variant="md" mb={1} textAlign="center">
-        Download the app to stay current as new artworks are available on Artsy.
+        Get alerts direct to your phone when you download the Artsy app.
```

Comment:
I think [here](https://www.figma.com/file/xsnfIWE7H67ZndGoEPVtN0/Alerts-Sandbox?node-id=1255%3A19002) it is specified to use mobile over phone, unless this has to do with the yesterdays discussion about the duplicate copy
```suggestion
        Get alerts direct to your mobile when you download the Artsy app.
```

Reply:
I copied the text from [this file](https://docs.google.com/document/d/1QexY-xIeQS8mH8cTjfCFUZYaSMveczsJmkYcAU7bX7g/edit#). But I'll check with Lois again just in case.

![image](https://user-images.githubusercontent.com/3513494/153185798-2b347b0a-6cea-4663-8b6f-b472fbc79d82.png)